source/release-notes/database-tools-changelog.txt

```diff
@@ -23,17 +23,16 @@ Database Tools |tool-version| Changelogs
 All Releases
 ------------
 
-.. include:: /includes/changelogs/dbtools-changelogs.rst
+.. include:: /includes/changelogs/releases/100.12.0.rst
```

Comment:
[reorder]

On line 26, I think we want `.. include:: /includes/changelogs/dbtools-changelogs.rst`.

On line, 32 I think we want `.. include:: /includes/changelogs/releases/100.12.0.rst`.

As things stand currently on staging, the **Database Tools 100.12.0** heading appears at the bottom of the page.

<img width="917" alt="Screenshot 2025-04-04 at 11 00 01 AM" src="https://github.com/user-attachments/assets/10849fdc-d707-43e5-adb6-93c2090ecdf0" />